numpy/polynomial/polynomial.py

```diff
@@ -1257,7 +1257,14 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None):
         chosen so that the errors of the products ``w[i]*y[i]`` all have the
         same variance.  When using inverse-variance weighting, use
         ``w[i] = 1/sigma(y[i])``.  The default value is None.
-
+    cov : bool, optional
+        If given and not False, return the covariance matrix. By default, 
+        the covariance are scaled by chi2/dof, where dof = M - (deg + 1),
```

Comment:
The chi2 refers to chi-squared, and is equal to

$$
\chi^2 = \sum_{k=0}^{N-1}w_k (y_k - f(\mathbf{p}, x_k)^2,
$$

where $w_k = 1/\sigma_k^2$ are the weights using the *standard definition*, as given at this [link](https://www.itl.nist.gov/div898/handbook/pmd/section4/pmd432.htm), for example. Currently, `polyfit` uses a *nonstandard* definition, $w_k = 1/\sigma_k$, which should also change (although it appears to have been this way since v1.5).

Reply:
Ugh. I see that the nonstandard definition of the weights has remained an open issue for nearly *ten years*: #5261.

Reply:
I notice this is a documentation-related problem, isn't it? From https://github.com/numpy/numpy/issues/5261, I took https://github.com/numpy/numpy/issues/5261#issuecomment-779308989 to complete the documentation. But I'm not sure if this is correct.